testcase.py

```diff
@@ -25,6 +25,8 @@ def __init__(self):
 
         '''
 
+        # Set BOPTEST version number
+        self.version = '0.0.0'
```

Comment:
should we put the "version" into a __init__.py?

Reply:
What's the advantage(s) of that?

Reply:
I thought this was somehow a standard? https://packaging.python.org/guides/single-sourcing-package-version/#single-sourcing-the-version

Reply:
I see thanks.  What about 4. on the list in the link you provide?  Since the BOPTEST repo does not just consist of Python modules and is not "installed" via pip.  We could create a version.txt in the root directory and have testcase.py read it on instantiation?

Reply:
looks good to me

Reply:
Ok, will use version.txt.

Reply:
Done in https://github.com/ibpsa/project1-boptest/pull/336/commits/8067815a5d588496c2fae80676142dfc74e3a769.